Mail/Mailer.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.IO;
+using System.Text;
+using MailKit.Net.Smtp;
+using MimeKit;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace NetCoreBootstrap.Mail
+{
+    public class Mailer
+    {
+        private readonly SmtpClient _smtpClient;
+        private string host, username, password, name, email;
+        private int hostPort;
+        private readonly string _jsonFilePath;
+        private readonly char _jsonSplitter = ':';
+        
+        public Mailer()
+        {
+            _smtpClient = new SmtpClient();
+            _jsonFilePath = "appsettings.Development.json";
```

Comment:
we need to put appsettings.Development.json or appsettings.json, dependening of the enviroment